Sources/AnimalKingdomAPI/GeneratedCCN/Operations/ClassroomPetsCCNQuery.swift

```diff
@@ -0,0 +1,70 @@
+// @generated
+// This file was automatically generated and should not be edited.
+
+import ApolloAPI
+
+public class ClassroomPetsCCNQuery: GraphQLQuery {
+  public let operationName: String = "ClassroomPetsCCN"
+  public let document: DocumentType = .notPersisted(
+    definition: .init(
+      """
+      query ClassroomPetsCCN {
+        classroomPets[!]? {
```

Comment:
List syntax example that uses both an element designator and a designator on the field as a whole.